gammapy/data/data_store.py

```diff
@@ -367,3 +368,44 @@ def _get_min_energy_threshold(observation_table, data_dir):
         min_energy_threshold = min(min_energy_threshold, energy_threshold)
 
     return min_energy_threshold
+
+
+class Observation(object):
+    """IACT observation dataset container.
+
+    TOOD: document
+
+    Examples
+    --------
+    Usually you'll use the `gammapy.data.DataStore` to load IACT `Observation` datasets like this:
+
+    >>> from gammapy.data import DataStore
+    >>> ds = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2')
+    >>> obs = ds.load_obs(obs_id=23037)
+    >>> obs.info()
+    TODO: prints summary output...
+    """
+    def __init__(self, events=None, gti=None, aeff=None, edisp=None, psf=None):
+        self.events = events
+        self.gti = gti
+        self.aeff = aeff
+        self.edisp = edisp
+        self.psf = psf
+
+    def peek(self):
+        """Quick-look plots in a few panels."""
+
+    def info(self):
+        """Print some summary info to stdout."""
+
+    def make_exposure_image(self, fov, energy_range):
+        """Make exposure image ...
+
+        Do we want such methods here or as standalone functions that work with obs objects?
+        """
+
+    def make_spectrum_observation(self, target_pos, on_radius):
```

Comment:
I usually but classmethods on my classes, i.e. in this case I would have put

SpectrumObservation.from_observation()

But I guess that's a matter of taste. I might be better to have make_object funktions because it's more intuitive from the users' point of view.